src/LodeRunner.Core/Models/LoadClient.cs

```diff
@@ -68,7 +68,7 @@ public class LoadClient : BaseEntityModel
         /// <returns>a new LoadClient entity.</returns>
         public static LoadClient GetNew(ILRConfig config, DateTime lastUpdated)
         {
-            return new LoadClient
+            var loadClient = new LoadClient
```

Comment:
we should be using the "new ()" syntax - make sure your style cop is up to date

Reply:
this is being missed because the nuget packages are out of date

Reply:
.editorconfig and stylecop.json are duplicated in the project - it's better to have exactly one of each and reference - you should be able to have each in the src directory

Reply:
@bartr, there are several different projects loaded for each solution.  I think there is one of those files for each project, but it means that when you open a solution file you would find them in each project.  We could move it to a single file for each that exists outside the project folder structure where we're storing the .sln files.  Would that be better?

Reply:
switched syntax

Reply:
@jofultz  that's where I would suggest putting the .editor and stylecop - that way you have "exactly one" - with the nuget upgrade, I think you're going to want to change some rules, so having one is goodness - the new "new ()" rule is painful to fix (assuming you adopt the rule)

Reply:
@alfredoihernandez - there are likely hundreds of the new () changes - style cop will catch them once you upgrade and config - team should decide if they are going to adopt the new style or keep the old one then resolve over weeks or a few months if you change. I think the new () syntax is a lot simpler personally - others don't like it. Consistency (and rule setup) are the keys.

Reply:
@alfredoihernandez - btw - you don't have to fix all of these in this PR - just add an issue(s) and resolve the conversation(s) - don't want to hold up the PR too long

Reply:
https://github.com/retaildevcrews/wcnp/issues/556